backend/search.js

```diff
@@ -385,6 +385,7 @@ class Search {
       // https://github.com/ryanhugh/searchneu/pull/19
       numerica: 'numerical',
       numeri: 'numerical',
+      swd: 'software dev',
```

Comment:
DJ is working on moving over core search to elasticsearch, drop a comment about `swd` here! https://github.com/ryanhugh/searchneu/pull/87/files#diff-d74b9dd40c18a857e1ddbac80c22003cR90